lib/unhangout-routes.js

```diff
@@ -788,6 +795,136 @@ module.exports = {
         // Starting and stopping events
         //
 
+        app.get("/followup/event/:id/participant_:participantCount", function(req, res) {
+
+            var event = db.events.get(req.params.id);
+
+            var eventTitle = event.get("title");
+            var history = event.get("history");
+            var participantCount = req.params.participantCount; 
```

Comment:
Change in variable name should propagate all the way through, to ensure this code is readable and self-documenting.